src/test/java/controllers/dialogs/ConfigDialogControllerTest.java

```diff
@@ -0,0 +1,141 @@
+package controllers.dialogs;
+
+import api.PSPortFactory;
+import api.PSPortSSL;
+import api.PSPortTCP;
+import com.jfoenix.controls.JFXButton;
+import com.jfoenix.controls.JFXComboBox;
+import com.jfoenix.controls.JFXTextField;
+import javafx.application.Platform;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.Label;
+import javafx.util.Callback;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import utils.JavaFXThreadingRule;
+
+import java.lang.reflect.Field;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertNull;
+import static org.easymock.EasyMock.expect;
+import static org.powermock.api.easymock.PowerMock.*;
+
+/**
+ * Created by Gorka Olalde on 8/6/16.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({Callback.class, PSPortTCP.class, PSPortSSL.class, PSPortFactory.class})
+public class ConfigDialogControllerTest{
+
+    private ConfigDialog dialog;
+    private Label headerLabel;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'headerLabel' could be replaced by a local variable.](https://www.codacy.com/app/POPBL6/front-end/file/2720643971/issues/source?bid=3316499&fileBranchId=3347636#l36)